ice/src/main/java/com/altinity/ice/cli/Main.java

```diff
@@ -367,6 +369,40 @@ void deleteTable(
     }
   }
 
+  @CommandLine.Command(name = "delete", description = "Delete Partition(s).")
+  void deletePartition(
+      @CommandLine.Option(names = "--namespace", description = "Namespace name", required = true)
+          String namespace,
+      @CommandLine.Option(names = "--table", description = "Table name", required = true)
```

Comment:
specifying ns & table via --namespace & --table is inconsistent with the rest of the commands: ice insert, describe, etc. all expect ns.table as an arg.